website/docs/MocksAndSpies.md

```diff
@@ -7,7 +7,9 @@ WebdriverIO comes with built in support for modifying network responses that all
 
 :::info
 
-This feature is currently only supported when running local tests on Chrome. It is planned to be supported on [Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1591389) and [Sauce Labs](https://saucelabs.com/) soon. If you encounter problems using it please file [an issue](https://github.com/webdriverio/webdriverio/issues/new/choose) and let us know!
+Note that using the `mock` command requires support for Chrome DevTools protocol.
+That support is given if you run tests locally in Chromium based browser or if
+you use a Selenium Grid v4 or higher. If you use a cloud vendor for spawning browsers, make sure your vendor also supports the Chrome DevTools protocol.
```

Comment:
Do we know which cloud vendor currently support DevTools in the cloud? I wonder if we can just say Sauce and Browserstack support it which would cover probably the majority of users.

Reply:
lambdatest too from what I understand